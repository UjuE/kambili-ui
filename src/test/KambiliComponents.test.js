import React from 'react';
import { shallow } from 'enzyme';
import MealTypeComponent from '../main/KambiliComponents';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe("MealTypeComponent", () => {
    it("should accept parameters", () => {
        const component = shallow(<MealTypeComponent value="Breakfast" />);
        expect(component.getElements()).toMatchSnapshot();
    });
});