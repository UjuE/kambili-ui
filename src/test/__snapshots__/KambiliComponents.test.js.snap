// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MealTypeComponent should render correctly 1`] = `
Array [
  <div
    className="menu-type"
  >
    Breakfast
  </div>,
]
`;
